source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -1236,69 +1334,115 @@ TEST_F(NiDAQmxDriverApiTests, AIVoltageChannel_CfgSampClkTimingAndAcquireData_Su
 
 TEST_F(NiDAQmxDriverApiTests, ChannelWithDoneEventRegistered_RunCompleteFiniteAcquisition_DoneEventResponseIsReceived)
 {
+  const auto FINITE_SAMPLE_COUNT = 10UL;
   create_ai_voltage_chan(0.0, 1.0);
   ::grpc::ClientContext reader_context;
   auto reader = register_done_event(reader_context);
   reader->WaitForInitialMetadata();
 
-  const auto FINITE_SAMPLE_COUNT = 10UL;
   cfg_samp_clk_timing(
-      create_cfg_samp_clk_timing_request(1000.0, Edge1::EDGE1_UNSPECIFIED, AcquisitionType::ACQUISITION_TYPE_FINITE_SAMPS, FINITE_SAMPLE_COUNT));
+      create_cfg_samp_clk_timing_request(1000.0, Edge1::EDGE1_RISING, AcquisitionType::ACQUISITION_TYPE_FINITE_SAMPS, FINITE_SAMPLE_COUNT));
```

Comment:
This fixes:
```
[ RUN      ] NiDAQmxDriverApiTests.ChannelWithDoneEventRegistered_RunCompleteFiniteAcquisition_DoneEventResponseIsReceived
unknown file: error: C++ exception with description "Requested value is not a supported value for this property. The property value may be invalid because it conflicts with another property.
Property: DAQmx_SampClk_ActiveEdge
Requested Value: 0
Possible Values: DAQmx_Val_Rising, DAQmx_Val_Falling

Task Name: _unnamedTask<1E>

Status Code: -200077" thrown in the test body.
[  FAILED  ] NiDAQmxDriverApiTests.ChannelWithDoneEventRegistered_RunCompleteFiniteAcquisition_DoneEventResponseIsReceived (31 ms)
```